linera-storage-service/Cargo.toml

```diff
@@ -22,9 +22,11 @@ name = "storage_service_server"
 path = "src/server.rs"
 
 [dependencies]
+anyhow = { workspace = true }
 async-lock = { workspace = true }
 async-trait = { workspace = true }
 clap = { workspace = true }
+linera-service = { workspace = true }
```

Comment:
We switched to the `linera-service.workspace = true` syntax in all the other `Cargo.toml`s.

Reply:
Ok.